### ISSUE TITLE
Change link on index.html

### ISSUE THREAD

**Issue by jywarren** [publiclab/community-toolbox#1]
### 🆕🐥☝ First Timers Only.

This issue is reserved for people who never contributed to Open Source before. We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

[About First Timers Only](http://www.firsttimersonly.com/).

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below.

- [ ] 📝 **Update** the file [index.html](https://github.com/publiclab/community-toolbox/blob/master/index.html) in the `community-toolbox` repository (press the little pen Icon) and edit the line as shown below.

```diff
@@ -28,8 +28,9 @@
       <header class="text-center">
         <h1>Welcome to Public Lab!</h1>
         <p>
-          Powered by Commmunity Toolbox
-          <a href="https://github.com/jywarren/community-toolbox"><i class="fa fa-github"></i></a> Open Source
+          Powered by <a href="https://github.com/jywarren/community-toolbox"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/jywarren/community-toolbox">Commmunity Toolbox</a> 
+          | Open Source
         </p>
       </header>
 
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/)

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions

Leave a comment below!

This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).

**Comment by jywarren** [comment]
Over here, @chocos10 !! Thanks!!

**Comment by MrGlacier** [comment]
i want to claim this issue :)

**Comment by phillipesouza** [comment]
I want to contribute too.

**Comment by jywarren** [comment]
Hi, everybody! I'm so glad you're interested! I just invited @chocos10, @MrGlacier and @phillipesouza. I'm sure we can come up with issues for everybody!

**Comment by fitchae** [comment]
I would like to claim this issue!

**Comment by vagita24** [comment]
Can I contribute to this issue...

**Comment by Strider1990** [comment]
I would like to claim this issue!

**Comment by denisenricohasyim93** [comment]
i would like to claim this issue...

**Comment by jywarren** [comment]
Hi, folks - please submit a PR for us; I'm modifying the steps accordingly!

**Comment by jywarren** [comment]
And apologies -- the process of inviting, accepting, adding people was a bit of a roadblock, so we're going to go ahead and just accept a PR from whoever is able to post one -- thank you!

**Comment by vagita24** [comment]
hi
@jywarren i have sent a pull request. please review it

**Comment by jywarren** [comment]
Thanks, @vagita24 -- solved in #4 -- super! THank you everyone, and please check out the issues list if you'd like to help out further! There are also more listed at:

https://publiclab.github.io/community-toolbox
